src/main/kotlin/lotto/domain/value/Money.kt

```diff
@@ -0,0 +1,26 @@
+package lotto.domain.value
+
+import java.math.BigDecimal
+import java.math.BigInteger
+
+data class Money(private val won: BigInteger) {
+    init {
+        if (won < BigInteger.ZERO) throw IllegalArgumentException("음수는 Money로 사용될 수 없습니다.")
+    }
+
+    operator fun plus(money: Money) = Money(won.plus(money.won))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    operator fun plus(money: Money) = Money(won + money.won)
```